docs/examples/example_figure.py

```diff
@@ -0,0 +1,83 @@
+from matplotlib import pyplot as plt
+from matplotlib import colors as colors
+import numpy as np
+from astropy import units as u
+from screens.fields import dynamic_field
+from screens.secspec import SecondarySpectrum
+from screens.visualization import make_sketch
+
+
+def gen_magnifs(th):
+
+    # Complex magnifications amplitudes of the scattering points
+    sig = 1.5*u.mas
+    a = 0.3*np.exp(-0.5*(th/sig)**2).to_value(1)
+    np.random.seed(2)
+    magnifs = a * np.random.normal(size=(th.size, 2)).view('c16').squeeze(-1)
+    magnifs[np.where(th == 0)] = 1
+    magnifs /= np.sqrt((np.abs(magnifs)**2).sum())
+
+    return magnifs
+
+
+def example_figs(th_par, magnifs,
```

Comment:
Maybe just name if `make_figure`, `simple_figure` or so? We're likely to have more figures in here soon... (can also call the file `simple_figure` and call the function `figure`)

Reply:
Done. These names may have to change when we have a better (idea of) the structure of the docs.